rocky/reports/templates/forms/report_form_fields.html

```diff
@@ -18,7 +18,7 @@
     {% endfor %}
 {% endif %}
 {% for required_optional_plugin, plugins_ in plugins.items %}
-    {% for plugin in plugins_ %}<input type="hidden" name="plugin" value="{{ plugin.id }}">{% endfor %}
+    {% for plugin in plugins_ %}<input type="hidden" name="all_plugins" value="{{ plugin.id }}">{% endfor %}
```

Comment:
Just to be sure: The views still seem to refer to the `plugin` query parameter, so I'm unsure if this change works. Regardless, I think `plugins` as a query parameter list would've been a better name

Reply:
Yeah weird right. It seems that this variable is and was not used... But I'm afraid to delete it.